addinstanceform.php

```diff
@@ -33,17 +36,26 @@
  * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 class cachestore_redis_addinstance_form extends cachestore_addinstance_form {
+    /** @var bool */
+    private static $clusteravailable = null;
+
+    protected static function is_cluster_available() {
```

Comment:
I made this static because it is not related to any instance of the cachestore, it is just used to detect (and mock if needed) if the Cluster PHP module is available.